### ISSUE TITLE
Standardize deps badges

### ISSUE THREAD

**Issue by alexdln** [npmx-dev/npmx.dev#628]
Dependencies have different badges - vulnerabilities, outdated, optional, and potentially others.

Currently, outdated is displayed as an icon, and optional as text (in a fairly large container). It's worth consolidating their appearance, and it would be more convenient to use icons (again, optional takes up too much space and attention).

@brenoepics did this for optional in his [PR](https://github.com/npmx-dev/npmx.dev/pull/549) and I think the icon really does look more convenient, but we haven't been able to choose a suitable icon yet and have rolled back this part and am creating a separate issue for choosing and solving it.

Example page with optional deps - [vitest](https://npmx.dev/vitest)

**Comment by Flo0806** [comment]
PR is merged - should be finished

**Comment by alexdln** [comment]
@Flo0806 Sorry, maybe I described it poorly in the issue, but I made some improvements in that PR and then created an additional task to standardize badges, since we couldn’t decide at that time how exactly it should look

This is an unification issue. It's not yet closed

<img width="458" height="182" alt="Image" src="https://github.com/user-attachments/assets/325c1f5d-2fd4-43e7-ac88-ab5807bda636" />

<img width="442" height="225" alt="Image" src="https://github.com/user-attachments/assets/f13363cd-2c58-434f-b019-1840c67dfaf5" />

<img width="373" height="140" alt="Image" src="https://github.com/user-attachments/assets/694a3eb3-7779-4046-a555-28f65cc978e0" />
